drf_tweaks/autodoc.py

```diff
@@ -170,6 +170,32 @@ def _generate_text(cls, documented_cls, method_name):
         return text
 
 
+class OnDemandFieldsAutodoc(AutodocBase):
+    @classmethod
+    def _generate_yaml(cls, documented_cls, method_name):
+        return ""
+
+    @classmethod
+    def _generate_text(cls, documented_cls, method_name):
+        serializer = getattr(documented_cls, "serializer_class", None)
+        if not serializer:
+            return ""
+
+        try:
+            on_demand_fields = serializer.Meta.on_demand_fields
+        except AttributeError:
+            on_demand_fields = set()
```

Comment:
Nope, the next condition is neccessary in case of `on_demand_fields=[]` set in serializer meta